feature/home/src/main/java/com/epmedu/animeal/home/presentation/ui/MapboxMap.kt

```diff
@@ -28,7 +34,10 @@ fun MapboxMap(state: HomeState) {
     LaunchedEffect(Unit) { initialLocationReceived = false }
 
     AndroidView(
-        factory = { mapBoxView },
+        factory = {
+            addMarkers(mapBoxView, state.feedingPoints)
```

Comment:
Do we have something like onClick listener for the markers? If so, can you please add a listener and pass it as an argument to MapboxMap

Reply:
Yeah no problem. I'm on it.